### ISSUE TITLE
install ruby on rails

### ISSUE THREAD

**Issue by lcabrini** [AluguntuguiProject/aluguntugui#11]
Most other issues depend on this ...

**Comment by benbaka** [comment]
It seems you will be needing node.js installed

**Comment by lcabrini** [comment]
Hm.. so rails now depends on node.js. Interesting...

**Comment by lcabrini** [comment]
OK, since it seems to work, I've moved it to testing. I'm good to move sign it off as completed. As that OK?

**Comment by lcabrini** [comment]
Man, my spelling is at an all-time low. "is that OK?", was the intended question in the last comment.

**Comment by benbaka** [comment]
Yes. I am ok with it. Just wondering how we are going to test that.

**Comment by lcabrini** [comment]
In this particular case, I think running the server and seeing the default RoR page is testing enough.

**Comment by benbaka** [comment]
I guess so. The smoke test will also do.

**Comment by lcabrini** [comment]
Result of smoke test: *cough, cough* :D

**Comment by Claudetteg1** [comment]
looking for a LOCAL Ruby/Python Developer for a client in Overland, KS.  This is a 12 month contract opportunity. Open to c2c candidates.
•	Must have a minimum of 2 years Ruby/Rails or Python development experience (Ruby is highly preferred) 
•	Strong understanding of JavaScript and RDBMS/SQL.
•	 At least 1 years’ experience within Unix/Linux environments, APIs, ERBs, jQuery, and Oracle

Please send resume to cgraham@tekpartners.com TODAY!
